concerto-behaviors.html

```diff
@@ -181,13 +181,19 @@
             this.setAttribute(key.toLowerCase(), JSON.stringify(data[key]));
           } else {
             this.setAttribute(key.toLowerCase(), data[key]);
+            // TODO! @gabe please look at this and see why the prior line is not setting all the base properties
```

Comment:
@mfrederickson It looks like the line `this[key] = data[key];` sets the content object's property instead of the attribute, but since we have `reflectToAttribute` [set to true](https://github.com/concerto/concerto-frontend/blob/master/concerto-behaviors.html#L130) then the `contentId` property is mapped to the `content-id` attribute [based on these rules](https://www.polymer-project.org/1.0/docs/devguide/properties#property-name-mapping) (scroll down to "Property name to attribute name mapping" if the link wasn't working well). 

After inspecting the content element using the developer console, I noticed we did have `contentid` and `content-id` attributes set. The former must've been set by our call to `this.setAttribute()` and the latter must've been set via object property being reflected to an attribute. 

It seems like `this[key] = data[key];` is not guaranteed to set everything as an attribute, unless `reflectToAttribute` is explicitly stated. 

After messing around with the code a little bit, I think it would be best if we set everything one specific way to avoid confusion. I found it easiest to set everything as a property like you did with `this[key] = data[key];` and then reflect the properties to attributes as needed with `reflectToAttribute`. I'm open to any other ideas though, just figured this part of the code could definitely use some cleaning up / refactoring.